server/schemas/user.schema.js

```diff
@@ -45,6 +45,17 @@ const vendor = Joi.object().keys({
   companyName: optionalString('Company Name'),
 });
 
+const updateVendor = Joi.object().keys({
```

Comment:
I believe this can be renamed as vendor.
```suggestion
const vendor = Joi.object().keys({
```

Reply:
there's already a `vendor` variable on line 44

Reply:
Understood and thanks for the heads up. However, I think this defines a vendor schema better that the one used for vendor. Let's change the name of the other one to `vendorCompany` or any associated name and use `vendor` for this 🙏🏽